tests/cli/test_toolkit.py

```diff
@@ -524,3 +524,15 @@ def test_get_relative_path_from_config_file_to_data_base_file_path_from_misc_dir
     )
     obs = get_relative_path_from_config_file_to_base_path(ge_dir, absolute_path)
     assert obs == os.path.join("..", "..", "data", "pipeline1")
+
+
+def test_get_default_expectation_suite_name_with_partial_batch_request_definition():
+    batch_request = {
+        "datasource_name": "snowflake",
+        "data_connector_name": "default_runtime_data_connector_name",
+        "limit": 1000,
+    }
```

Comment:
This is the exact `batch_request` that is passed to `get_default_expectation_suite_name` from the CLI error:

<img alt="Screenshot 2023-02-20 at 16 36 17" src="https://user-images.githubusercontent.com/1042275/220160954-fa7aaff9-9930-4bb9-abd9-f67b4b5454fd.png">